tk_toolchain/cmd_line_tools/tk_build_resource/__init__.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+
+import subprocess
+import argparse
+import re
+
+
+def process_import_line(module, import_text):
+    return (
+        f'from {import_text} import {module}\n'
+        f'for name, cls in {module}.__dict__.items():\n'
+        f'    if isinstance(cls, type): globals()[name] = cls\n'
```

Comment:
Maybe use triple-quote multi-line string to improve readability?

Reply:
I think that is better in readability now: because with triple quotes would be like:
```
def process_import_line(module, import_text):
    return f"""from {import_text} import {module}
for name, cls in {module}.__dict__.items():
    if isinstance(cls, type): globals()[name] = cls
"""
```